awesomplete.js

```diff
@@ -39,7 +39,12 @@ var _ = function (input, o) {
 		item: _.ITEM,
 		replace: _.REPLACE,
 		tabSelect: false,
-		listLabel: "Results List"
+		listLabel: "Results List",
+		tStatusResult: "${length} results found",
+		tNoResults: "No results found",
+		tStatusQueryTooShort: "Type ${minChars} or more characters for results.",
+		tStatusStartTyping: "Begin typing for results.",
+		tListItemText: "list item ${index} of ${length}"
```

Comment:
I think these should be in a separate object, not just as raw properties. Also, we do not need them on every instance, what we need is a way to select a language for the instance. So I'd suggest a data structure like: 

```js
Awesomplete.languages = {
	"en-us": {
		"list-label": "Results List"
	}
}
```

Then instances would point to one of these objects either by storing the language, or a reference to the object (I'll leave that up to you).